TypeCobol/Compiler/Nodes/Node.cs

```diff
@@ -643,6 +643,35 @@ private void Dump(StringBuilder str, int i)
         public bool IsInsideCopy() {
             return CodeElement != null && CodeElement.IsInsideCopy();
         }
+
+        public IDictionary<StorageArea, Tuple<string,DataDefinition>> StorageAreaReadsDataDefinition { get; internal set; }
+        public IDictionary<StorageArea, Tuple<string,DataDefinition>> StorageAreaWritesDataDefinition { get; internal set; }
+
+        public IEnumerable<Tuple<string,DataDefinition>> GetVariableFromNodeStorageArea(StorageArea storageArea,
```

Comment:
As StorageArea is the key, why use its SymbolReference to the associated value ?

Reply:
It was the way a DataDefinition was returned from the SymbolTable. I updated the search using directly the StorageArea key from the dictionary.